man/loadOutput.Rd

```diff
@@ -0,0 +1,49 @@
+% Generated by roxygen2 (4.1.0): do not edit by hand
```

Comment:
@jiwalker-usgs R is kind of weird this way, because we are using a tool to auto-build them. They used to not be created in tandem w/ the build. 

I can't think of a good reason to commit them except that I am guessing `devtools::install_github()` wouldn't give you documentation if the `Rd` files weren't on the repo. Not sure though.